File: ApplicationServer/src/main/java/io/github/michaelbui99/manhwatrackerapplicationserver/model/manhwalist/ManhwaListEntry.java
package io.github.michaelbui99.manhwatrackerapplicationserver.model.manhwalist;

import io.github.michaelbui99.manhwatrackerapplicationserver.model.manhwa.Manhwa;

public class ManhwaListEntry {
    private int id;
    private Manhwa entry;

    public ManhwaListEntry(int id, Manhwa entry) {
        this.id = id;
        this.entry = entry;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Manhwa getEntry() {
        return entry;
    }

    public void setEntry(Manhwa entry) {
        this.entry = entry;
    }
}

File: ApplicationServer/src/ManhwaTrackerApplicationServer/Models/Domain/ManhwaList/ManhwaListEntry.cs
using System.ComponentModel.DataAnnotations;

namespace ManhwaTrackerApplicationServer.Models.ManhwaList;
using Manhwa;

public class ManhwaListEntry
{
    [Key]
    public int Id { get; set; }
    public Manhwa Manhwa { get; set; }
    public int LatestReadChapter { get; set; }
}